### PR TITLE
kk_KZ Company/person identification numbers unit tests

### PR DESCRIPTION
**Pull request by YerlenZhubangaliyev**
Some unit tests

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
+.idea
```
Comment (fzaninotto) [inline .gitignore]: no, please remove

File: src/Faker/Provider/kk_KZ/Company.php [hunk 2]
```diff
@@ -52,7 +52,7 @@ public static function companyNameSuffix()
     /**
      * National Business Identification Numbers
      *
-     * @link   http://egov.kz/wps/portal/!utWCM/p/b1/04_Sj9Q1MjAwsDQ1s9CP0I_KSyzLTE8syczPS8wB8aPM4oO8PE2cnAwdDSxMw4wMHE08nZ2CA0KDXcwMgQoikRUYWIY4gxS4hwU4mRkbGBgTp98AB3A0IKQ_XD8KVQkWF4AV4LHCzyM_N1U_uKhUPzcqx83SU9cRANth_Rk!/dl4/d5/L0lHSkovd0RNQU5rQUVnQSEhLzRKVUUvZW4!/
+     * @link   http://egov.kz/wps/portal/Content?contentPath=%2Fegovcontent%2Fbus_business%2Ffor_businessmen%2Farticle%2Fbusiness_identification_number&lang=en
```
Comment (fzaninotto) [inline src/Faker/Provider/kk_KZ/Company.php]: can you explain this change?

File: test/Faker/Provider/kk_KZ/PersonTest.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<?php
+namespace Faker\Test\Provider\kk_KZ;
+
+use Faker\Generator;
+use Faker\Provider\kk_KZ\Person;
+
+/**
+ * Person test
```
Comment (fzaninotto) [inline test/Faker/Provider/kk_KZ/PersonTest.php]: useless comment

File: test/Faker/Provider/kk_KZ/CompanyTest.php [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<?php
+namespace Faker\Test\Provider\kk_KZ;
+
+use Faker\Generator;
+use Faker\Provider\kk_KZ\Company;
+
+/**
+ * Company test
```
Comment (fzaninotto) [inline test/Faker/Provider/kk_KZ/CompanyTest.php]: useless comment

File: test/Faker/Provider/kk_KZ/CompanyTest.php [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+     */
+    public function setUp()
+    {
+        $this->faker = new Generator();
+
+        $this->faker->addProvider(new Company($this->faker));
+    }
+
+    /**
+     * @see \Faker\Provider\kk_KZ\Person::businessIdentificationNumber
```
Comment (fzaninotto) [inline test/Faker/Provider/kk_KZ/CompanyTest.php]: useless comment

File: test/Faker/Provider/kk_KZ/PersonTest.php [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+     */
+    public function setUp()
+    {
+        $this->faker = new Generator();
+
+        $this->faker->addProvider(new Person($this->faker));
+    }
+
+    /**
+     * @see \Faker\Provider\kk_KZ\Person::individualIdentificationNumber
```
Comment (fzaninotto) [inline test/Faker/Provider/kk_KZ/PersonTest.php]: useless comment

File: src/Faker/Provider/kk_KZ/Company.php (hunk 2 above)
Comment (YerlenZhubangaliyev) [inline src/Faker/Provider/kk_KZ/Company.php]: Old link is outdated and wrong

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**